content/docs/learning-guide/tutorials/_index.md

```diff
@@ -41,6 +41,7 @@ Do you have O3DE installed and you aren't sure where to start? How about creatin
 | Tutorial | Description |
 | - | - |
 | [Create a Level](environments/create-a-level) | Create a level for O3DE. |
+| [Generate Terrain](environments/generate-terrain) | Learn to generate terrain from image gradients with entities or **Landscape Canvas**. |
```

Comment:
I'd like to keep a more verbose description for SEO and discoverability.

Changed to:

`| [Create Terrain](environments/create-terrain) | Learn to create terrain from images with entities or **Landscape Canvas**. |`

Reply:
Hmmm. How about this? "Learn to create terrain from images with **Landscape Canvas** or direct entity creation." 

I'm probably over-thinking this, but I keep struggling with the semantics of "with entities or Landscape Canvas" since Landscape Canvas still uses entities.

Reply:
Alright, we'll just go with the original suggestion.